packages/react-storage/src/components/StorageBrowser/Views/Controls/Table.tsx

```diff
@@ -214,3 +241,95 @@ export const LocationsViewTable = (): JSX.Element => {
     />
   );
 };
+
+export const LocationDetailViewTable = (): JSX.Element => {
```

Comment:
Should I move these view specific tables into a different file? The only issue if I do that is it would require the table components to be exported. Does anyone have any thoughts on this?